File: scripts/java/billion.java
public class billion {
    public static void main(String[] args) {
        int billion = 1000000000;
        long startTime = System.currentTimeMillis();

        for (int i = 0; i < billion; i++) {
        }

        long endTime = System.currentTimeMillis();
        double elapsedTime = (endTime - startTime) / 1000.0; // segundos
        System.out.printf("%.3f segundos\n", elapsedTime);
    }
}

File: scripts/php/billion.php
<?php
$billion = 1000000000;

$start_time = microtime(true);

for ($i = 0; $i < $billion; $i++) {
}

$end_time = microtime(true);

$elapsed_time = $end_time - $start_time;
printf("%.3f segundos\n", $elapsed_time);
?>
